src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader<T> {
+    List<T> parseDataFrom(String fileName);
```

Comment:
it's reader, not parser
```suggestion
public interface FileReader {
    List<String> readDataFrom(String fileName);
```